blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Version v0.22 release"
+linkTitle: "Version v0.22 release"
+Author: "[Carlos Santana](https://twitter.com/csantanapr)"
+Author handle: https://github.com/csantanapr
+date: 2021-04-06
+description: "Knative v0.22 release announcement"
+type: "blog"
+image: knative-eventing.png
+---
+
+
+### Announcing Knative v0.22 Release
+
+A new version of Knative is now available across multiple components.
+Follow the instructions in the documentation [Installing Knative](https://knative.dev/docs/install/) for the respective component.
+
+#### Table of Contents
+- [Serving v0.22](#serving-v022)
+- [Eventing v0.22](#eventing-v022)
+- Eventing Extensions
+    - [Eventing Kafka Broker v0.22](#eventing-kafka-broker-v022)
+- `kn` [CLI v0.22](#client-v022)
+    - `kn` [CLI Plugins](#cli-plugins)
+- [Operator v0.22](#operator-v022)
+- [Thank you contributors](#thank-you-contributors)
+
+
+
+### Highlights
+
+- Serving Domain Mapping improves multi-tenency support to avoid domain name 'squatting' in cluster.
+- Eventing now allows Trigger users to set another namespace's Subscriber
+- Eventing kafka broker now Kubernetes's minimum version is 1.18
+- Eventing kafka now supports the ability to choose between ordered and unordered delivery
+- The CLI `kn` 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+- There are two new CLI plugins align with 0.22 release, [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) and [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka)
```

Comment:
```suggestion
- Version 0.22 of the `kn` CLI contains bug fixes and minor feature enhancements.
- A breaking change was required to align the client API with the Kubernetes client API.
- There are two new CLI plugins available in version 0.22; [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) and [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka).
```

Reply:
@omerbensaadon @snneji @RichardJJG can we choose one way to consistently refer to "version" across docs and Knative in general? Do you have any preference? I don't want to keep using "version 0.22" in some places and "v0.22" in others for example if we can help it. Can you maybe take this to the TOC, Omer?

Reply:
My opinion is that deciding to use 0.22 vs. v0.22 in the website/docs, is something that the docs/ux team can make the decision and document that this is the style across the website.

I added the question in the new doc that @omerbensaadon started https://docs.google.com/document/d/1tigolRzKhronvgoAJfszE-XHWWd0LQ8o_j0D73O9KvM/edit

Reply:
I'm making everything consistent for now to use `v0.22`